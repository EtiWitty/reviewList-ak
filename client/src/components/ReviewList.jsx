class ReviewList extends React.Component {
    constructor () {
        super ();
    }
    
    render() {
        return (
            <div>
                Hello World!
            </div>
        );
    }
}

export default ReviewList;