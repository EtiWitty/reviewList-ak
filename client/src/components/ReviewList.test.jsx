import React from 'react';
import {shallow} from 'enzyme';

import ReviewList from './ReviewList.jsx';

describe('ReviewList', () => {
    test('Rendering the simple component', () => {
        const wrapper = shallow (
            <ReviewList />
        );
        expect(wrapper).toMatchSnapshot();
    });  
});