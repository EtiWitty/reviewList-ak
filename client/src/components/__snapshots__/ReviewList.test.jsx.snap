// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewList Rendering the Review List component 1`] = `ShallowWrapper {}`;
